source/containers.tex

```diff
@@ -13253,9 +13253,9 @@
   inline constexpr sorted_unique_t sorted_unique{};
 
   template<class Key, class T, class Compare, class KeyContainer, class MappedContainer,
-           class Allocator>
+           class Alloc>
```

Comment:
Could you please check if linebreaks can now be removed?

Reply:
This has been done (since November). Ping?